Modules/nismodule.c

```diff
@@ -388,7 +427,7 @@ nis_maplist (char *dom)
 }
 
 static PyObject *
-nis_maps (PyObject *self, PyObject *args, PyObject *kwdict)
+nis_maps (PyObject *module, PyObject *args, PyObject *kwdict)
```

Comment:
`self` need be rename?

Reply:
Quite confusing when calling `get_nis_state` by named self.